examples/vision/siamese.py

```diff
@@ -0,0 +1,210 @@
+
+"""
+# Siamese Network with Contrastive Loss
+
+Author: [Mehdi](https://github.com/s-mrb)<br>
+Date created: 2021/05/02<br>
+Last modified: 2020/05/03<br>
+Description: Similarity Learning Using Siamese Network with Contrastive Loss
+"""
+
+"""
+## Introduction
+
+The [Siamese Network](https://papers.nips.cc/paper/1993/file/288cc0ff022877bd3df94bc9360b9c5d-Paper.pdf) introduced by Bromley and LeCun to solve signature
```

Comment:
This line is too long

Reply:
Do I have to keep 80 character length in rendered page or as raw text, I am confused whether characters in hyperlinks count or not.
Thanks

Reply:
(I'm going through the original paper and your text, will help with the structure/syntax.)

Reply:
The 80 char limit per line is nice because it helps with readability, I think. In this example, you could try something like:

```
The [Siamese network](https://papers.nips.cc/paper/1993/file/288cc0ff022877bd3df94bc9360b9c5d-Paper.pdf)
new line...
new line...
....
```

Reply:
agree with @8bitmp3 , the limit is for code, consider taking her(his?) advice

Reply:
Done!